### DISCUSSION TITLE
Student Benefits Taking 3 Months

### DISCUSSION THREAD

**Discussion by JaviZamudio** [community/community#134483] [GitHub Education]
### Hi everyone,

I submitted my application on may and was accepted, it said to wait a bit until my benefits activate. Reading here, it said 3-5 days, but it's been months now. I'm on a new semester and with school projects, i was just hoping to get the benefits before the load gets heavier. Thanks! ;)

**Comment by Irissuu** [comment]
check your email for any updates, go to the [github education page](https://education.github.com/pack) and try logging in there to see if you can access the benefits, if you still haven't received any instructions, reach out to github support. You can contact them through their [support page](https://support.github.com/). I believe the delay is due to the high demand for this benefit

**Comment by prabhjotschugh** [comment]
Hi @JaviZamudio 

It usually takes a few days, but sometimes it can take a little longer, even a couple of weeks; the waiting time may vary depending on different factors, such as the volume of requests.

When the benefits become available, the status label will turn purple and change to:

Congrats! Your request for @JaviZamudio was approved and your benefits are now available.
For further reference you can look at these discussions: https://github.com/orgs/community/discussions/132790 , https://github.com/orgs/community/discussions/131665 , https://github.com/orgs/community/discussions/132470 , https://github.com/orgs/community/discussions/133531, https://github.com/orgs/community/discussions/134279, https://github.com/orgs/community/discussions/135145

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

[How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
[How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)
